driver/src/main/java/oracle/nosql/driver/ops/ReplicaStatsRequest.java

```diff
@@ -0,0 +1,227 @@
+import oracle.nosql.driver.NoSQLHandleConfig;
+import oracle.nosql.driver.iam.SignatureProvider;
+import oracle.nosql.driver.ops.serde.Serializer;
+import oracle.nosql.driver.ops.serde.SerializerFactory;
+import oracle.nosql.driver.values.TimestampValue;
+
+/**
+ * Cloud service only.
+ * <p>
+ * Represents the argument of a {@link NoSQLHandle#getReplicaStats} operation
+ * which returns stats information for one, or all replicas of a replicated
+ * table, returned in {@link ReplicaStatsResult}. This information includes a
+ * time series of replica stats, as found in
+ * {@link ReplicaStatsResult.ReplicaStats}.
+ * <p>
+ * It is possible to return a range of stats records or, by default, only the
+ * most recent stats records if startTime is not specified. Replica stats
+ * records are created on a regular basis and maintained for a period of time.
+ * Only records for time periods that have completed are returned so that a user
+ * never sees changing data for a specific range.
+ * @see NoSQLHandle#getReplicaStats
+ *
+ * @since 5.4.13
+ */
+public class ReplicaStatsRequest extends Request {
+
+    private String replicaName;
+    private long startTime;
+    private int limit;
+
+    /**
+     * Sets the table name to use for the operation.
+     *
+     * @param tableName the name
+     *
+     * @return this
+     */
+    public ReplicaStatsRequest setTableName(String tableName) {
+        super.setTableNameInternal(tableName);
+        return this;
+    }
+
+    /**
+     * Sets the replica name to query the stats information. If not set,
+     * information for all replicas is returned.
+     *
+     * @param replicaName the replica name
+     *
+     * @return this
+     */
+    public ReplicaStatsRequest setReplicaName(String replicaName) {
+        this.replicaName = replicaName;
+        return this;
+    }
+
+    /**
+     * Gets the replica name to query the stats information or null if
+     * not set
+     *
+     * @return replica name
+     */
+    public String getReplicaName() {
+        return replicaName;
+    }
+
+    /**
+     * Sets the start time to use for the request in milliseconds since the
+     * Epoch in UTC time. If no start time is set for this request the most
+     * recent complete stats records are returned, the number of records is
+     * up to limit {@link ReplicaStatsRequest#setLimit}
+     *
+     * @param startTime the start time
+     *
+     * @return this
+     */
+    public ReplicaStatsRequest setStartTime(long startTime) {
+        if (startTime < 0) {
+            throw new IllegalArgumentException("startTime must be >= 0");
+        }
+        this.startTime = startTime;
+        return this;
+    }
+
+    /**
+     * Sets the start time from an ISO 8601 formatted string. If timezone
+     * is not specified it is interpreted as UTC.
+     *
+     * @param startTime the string of a Timestamp in ISO 8601 format
+     * "uuuu-MM-dd['T'HH:mm:ss[.f..f]]".
+     * @return this
```

Comment:
maybe add that this method will throw an IAE if the string is not correctly formatted